supplemental_notebooks/2020-07-30_pathway_analysis.Rmd

```diff
@@ -636,7 +636,7 @@ fit2 <- eBayes(fit)
 ```
 
 Let's take a look at the top 10 pathways returned.
-Without specifying a coefficient, `topTable()` will report the F-tests statistics for each of the three contrasts: treatment in Group 3, treatment in Group 4, and differences in response between the subtypes.
+Without specifying a coefficient, `topTable()` will report the F-statistics for testing whether any of the contrasts (treatment in Group 3, treatment in Group 4, and differences in response between the subtypes) are non-zero.
```

Comment:
This sounds better. I'm still not sure (from ignorance) whether this is what it does, or if it is more general, but I this seems fine.

Reply:
For posterity, from _13.1 Summary Top-Tables_

> The `eBayes()` function computes one more useful statistic.  The moderated F-statistic (F) combines  the t-statistics  for  all  the  contrasts  into  an  overall  test  of  significance  for  that  gene. The F-statistic tests whether any of the contrasts are non-zero for that gene, i.e., whether that gene is differentially expressed on any contrast.  The denominator degrees of freedom is the same as that of the moderated-t.  Its p-value is stored as `fit$F.p.value`.  It is similar to the ordinary F-statistic from analysis of variance except that the denominator mean squares are moderated across genes.

Reply:
👍 But I bet it was doing the other thing, due to the bug... (`fit` didn't know about the contrasts yet!)

Reply:
Yep this makes considerably more sense with the 🐛 fix!